journal.py

```diff
@@ -43,56 +53,99 @@
 SELECT_SINGLE_ENTRY = """
 SELECT * FROM entries WHERE id=%s;
 """
+SELECT_SINGLE_NEWEST_ENTRY = """
+SELECT * FROM entries ORDER BY created DESC LIMIT 1 
+"""
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+Base = declarative_base()
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+    title = sa.Column(sa.Unicode(127), nullable=False)
+    text = sa.Column(sa.UnicodeText, nullable=False)
+    created = sa.Column(
+        sa.DateTime, nullable=False, default=datetime.datetime.utcnow
+    )
+
+    def __repr__(self):
+        return u"{}: {}".format(self.__class__.__name__, self.title)
+
+    @classmethod
+    def all(cls):
+        return DBSession.query(cls).order_by(cls.created.desc()).all()
+
+    @classmethod
+    def by_id(cls, id):
+        return DBSession.query(cls).filter(cls.id==id).one()
 
+    @classmethod
+    def from_request(cls, request):
+        title = request.params.get('title', None)
+        text = request.params.get('text', None)
+        created = datetime.datetime.utcnow()
+        new_entry = cls(title=title, text=text, created=created)
+        DBSession.add(new_entry)
+
+    @classmethod
+    def get_newest(cls):
+        return DBSession.query(cls).order_by(cls.created.desc()).first()
 
 logging.basicConfig()
 log = logging.getLogger(__file__)
 
+def to_json(entry):
```

Comment:
could this be defined as a method of the entry class?  Then you could just call something like `instance.to_json()` and get what you wanted.